pkgs/servers/search/elasticsearch/plugins.nix

```
@@ -18,6 +18,9 @@ let
       inherit installPhase;
       pname = "elasticsearch-${pluginName}";
       dontUnpack = true;
+      # Work around the "unpacker appears to have produced no directories"
+      # case that happens when the archive doesn't have a subdirectory.
+      setSourceRoot = "sourceRoot=`pwd`";
```

```suggestion
      setSourceRoot = "sourceRoot=$(pwd)";
```

I generally don't prefer to use backticks https://stackoverflow.com/questions/9449778/what-is-the-benefit-of-using-instead-of-backticks-in-shell-scripts